README.md

```diff
@@ -158,6 +158,16 @@ Turn off colorized nicks:
 /set plugins.var.python.slack_extension.colorize_nicks 0
 ```
 
+Turn off colorized nicks in buffers.pl:
+```
+/set plugins.var.python.slack_extension.disable_color_nicks_in_side_buffer 1
```

Comment:
What about making these options positive, not negative and named "colorize_nicks_in_x" to make it feel more cohesive with the existing colorize option?